jfrog_platform_self-hosted/README.md

```diff
@@ -0,0 +1,274 @@
+## Overview
+
```

Comment:
@estherk15 doesn't updating the README update https://docs.datadoghq.com/integrations/jfrog_platform as well? If that is the case, linking it might confuse the customers

Reply:
Instead, I can add 

<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will be deprecated by July 31st 2023.</div>

Because we're using Fluentd and the check that is in place right now (JFrog Platform (self-hosted)) check will be removed by the end of this month.

Reply:
Thanks for the clarification, I may have been confused on what is being deprecated! 
Suggested wording: "Agent check will deprecate on July 31, 2023."

Reply:
changed it, can you please review again

Reply:
Thanks @MahithaB, just made a suggestion for formatting, approved for documentation!